wolfssl/wolfio.h

```diff
@@ -615,7 +615,81 @@ WOLFSSL_API void wolfSSL_SetIOWriteFlags(WOLFSSL* ssl, int flags);
     WOLFSSL_API   int wolfSSL_SetIO_LwIP(WOLFSSL* ssl, void *pcb,
                                 tcp_recv_fn recv, tcp_sent_fn sent, void *arg);
 #endif
+#ifdef WOLFSSL_ISOTP
+    #define ISOTP_DEFAULT_TIMEOUT 100
+    #define ISOTP_DEFAULT_WAIT_COUNT 3
+    #define ISOTP_DEFAULT_BUFFER_SIZE 16384
```

Comment:
I only see this `ISOTP_DEFAULT_BUFFER_SIZE` referenced in the example docs. Can you comment on how you came up with this? Is it the max TLS packet size?

Reply:
"yes", but this actually got me thinking. ISO-TP cannot send more than 4095 bytes, which means it will never receive more than 4095 bytes. I ran some tests on the boundary here and found/fixed a couple of bugs (as well as fixing this definition).